### PR TITLE
Bp/verify consistency

### LINKED ISSUE #373: verify_consistency bug
Video: Lodo/2017-01-14_23-27-verify_consistency.mp4

### PR DESCRIPTION
**Pull request by Baxxx**
Closes #373

### CODE REVIEW

**Comment by sls-lnluksa** [comment]
Remove all logic around head voucher (marking voucher as head.)
For salary consistency only use:
1. Number of lines should be the same.
2. Sum on each account used in salary should be the same as the sum on journal.

**Review by sls-lnluksa — APPROVED** [review]
NOTE:
It will double line sum difference if we have 1 line with same amount but with different account plan on journal and salary. In other cases difference seems to be OK

Other than that, it is much better consistency check than before.

File: modules/report/view/verify_consistency.php [hunk 1]
```diff
@@ -663,7 +713,10 @@ function hovedbokreskonrooppdateringdiff() {
         <td><? print $salary->PayDate; ?></td>
         <td><? print $salary->DomesticBankAccount; ?></td>
         <td><? print $salary->KommuneNumber . " " . $salary->KommuneName; ?></td>
-        <td>Linjer avvike</td>
+        <td>
+            <? print $salary->LineDifference ? "There is line difference of <span style='color: red;'>". $salary->LineDifference ."</span style='color: red;'>. " : "" ?>
+            <? print $salary->SumDifference ? "There is sum difference of <span style='color: red;'>". $_lib['format']->Amount($salary->SumDifference) ."</span style='color: red;'>. " : "" ?>
```
Comment (Baxxx) [inline modules/report/view/verify_consistency.php]: Print 0 if necessary

**Review by stoivo — APPROVED** [review]

File: modules/report/view/verify_consistency.php [hunk 2]
```diff
@@ -663,7 +713,10 @@ function hovedbokreskonrooppdateringdiff() {
         <td><? print $salary->PayDate; ?></td>
         <td><? print $salary->DomesticBankAccount; ?></td>
         <td><? print $salary->KommuneNumber . " " . $salary->KommuneName; ?></td>
-        <td>Linjer avvike</td>
+        <td>
+            <? print "There is line difference of <span style='color: red;'>". $salary->LineDifference ."</span style='color: red;'>. " ?>
```
Comment (stoivo) [inline modules/report/view/verify_consistency.php]: Det er forskjell antall linjer

File: modules/report/view/verify_consistency.php [hunk 3]
```diff
@@ -663,7 +713,10 @@ function hovedbokreskonrooppdateringdiff() {
...
+            <? print "There is sum difference of <span style='color: red;'>". $_lib['format']->Amount($salary->SumDifference) ."</span style='color: red;'>. " ?>
```
Comment (stoivo) [inline modules/report/view/verify_consistency.php]: `Det er forskjell p&aring; belp&oslash; `

File: modules/report/view/verify_consistency.php (hunk 2 above)
Comment (agvald) [reply modules/report/view/verify_consistency.php]: Det er forskjellig antall linjer

**Pull request merged**